pkg/scheduler/apis/config/types.go

```diff
@@ -35,6 +40,16 @@ const (
 
 	// SchedulerDefaultProviderName defines the default provider names
 	SchedulerDefaultProviderName = "DefaultProvider"
+
+	// MaxTotalPriority defines the max total priority value.
+	MaxTotalPriority = int64(math.MaxInt64)
+	// MaxPriority defines the max priority value.
+	MaxPriority = 10
+	// MaxWeight defines the max weight value.
+	MaxWeight = MaxTotalPriority / MaxPriority
+	// DefaultPercentageOfNodesToScore defines the percentage of nodes of all nodes
+	// that once found feasible, the scheduler stops looking for more nodes.
+	DefaultPercentageOfNodesToScore = 50
```

Comment:
I think we shouldn't make those part of the API, I suggest we move them somewhere else internal, otherwise we will be stuck with supporting them.

Reply:
I suppose you meant to comment this on `staging/src/k8s.io/kube-scheduler/config/v1/types.go`? (`pkg/scheduler/apis/config/types.go` is for internal usage.)

Reply:
either both places or just staging. It depends on what we plan to do with the extender API as well.